test/dotnet-uninstall.Tests/Shared/Filterers/MajorMinorOptionFiltererTests.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System.Collections.Generic;
+using System.CommandLine;
+using Microsoft.DotNet.Tools.Uninstall.Shared.BundleInfo;
+using Microsoft.DotNet.Tools.Uninstall.Shared.Configs;
+using Microsoft.DotNet.Tools.Uninstall.Shared.Exceptions;
+using Xunit;
+
+namespace Microsoft.DotNet.Tools.Uninstall.Tests.Shared.Filterers
+{
+    public class MajorMinorOptionFiltererTests : FiltererTests
+    {
+        internal override Option Option => CommandLineConfigs.UninstallMajorMinorOption;
+        internal override string DefaultArgValue => "2.2";
+        internal override bool TestBundleTypeNotSpecifiedException => false;
+
+        public static IEnumerable<object[]> GetDataForTestFiltererGood()
```

Comment:
I still hold that such a name would be enough to communicate the meaning of the data, because the parameter names are in the signature of the actual testing method.

See https://github.com/dotnet/cli/blob/6da8a92712d114af8131c2683c0c0f78436f6d80/test/Microsoft.DotNet.Cli.Utils.Tests/StreamForwarderTests.cs.

Reply:
In ForwardingTheoryVariations, at least before the comma it is the input and after the comma it is the expectation. And in this PR case, it is hard to find what is the input

Reply:
Done.